Documentation/technical/commit-graph-format.txt

```diff
@@ -17,6 +17,9 @@ metadata, including:
 - The parents of the commit, stored using positional references within
   the graph file.
 
+- The Bloom filter of the commit carrying the paths that were changed between
```

Comment:
This commit message is also primarily RFC notes. Perhaps this could be combined with a Bloom filter data documentation?